homeassistant/components/surepetcare/const.py

```diff
@@ -13,6 +13,9 @@
 # sure petcare api
 SURE_API_TIMEOUT = 60
 
+# device info
+SURE_MANUFACTURER = "Sure Petcare"
+
```

Comment:
?

Reply:
Nope, sorry, thats from the implementation as devices (device integration? dunno how its called^^), one moment pls...